lib/pa_ess/utilities.ex

```diff
@@ -506,4 +506,170 @@ defmodule PaEss.Utilities do
+    {"Medford", "614"},
+    {"Wakefield Ave", "621"},
+    {"Quincy Center", "620"},
+    {"River St & Milton St", "619"},
+    {"Quincy Ctr", "620"},
+    {"Dudley", "632"},
+    {"Assembly", "4209"},
+    {"Wellington", "4024"},
+    {"Back Bay", "4036"},
+    {"Georgetowne", "674"},
+    {"Cleary Sq", "670"},
+    {"Reservoir", "4076"},
+    {"Wren St", "698"},
+    {"Dedham Line", "671"},
+    {"Dedham Sq", "847"},
+    {"Legacy Place", "848"},
+    {"Walpole Center", "697"},
+    {"East Walpole", "673"},
+    {"Stimson St", "693"},
+    {"Dedham Mall", "672"},
+    {"Charles River Loop", "669"},
+    {"Rivermoor", "677"},
+    {"VA Hospital", "696"},
+    {"Baker & Vermont St", "668"},
+    {"Columbian Sq", "805"},
+    {"Montello", "806"},
+    {"South Shore Plaza", "808"},
+    {"LaGrange St & Corey St", "675"},
+    {"LaGrange & Corey", "675"},
+    {"Townsend & Humboldt", "694"},
+    {"Union Sq", "695"},
+    {"Logan Airport", "562"},
+    {"Boston Medical Center", "568"},
+    {"Brighton Center", "566"},
+    {"Louis Pasteur Ave", "567"},
+    {"Ave Louis Pasteur", "567"},
+    {"Avenue Louis Pasteur", "567"},
+    {"Longwood Ave", "567"},
+    {"Nubian", "812"},
+    {"Adams & Gallivan", "569"},
+    {"Waltham", "561"},
+    {"Haymarket", "4028"},
+    {"Silver Line Way", "570"},
+    {"Drydock", "571"}
+  ]
+
+  @route_take_lookup %{
+    "SL5" => "587",
+    "SL4" => "586",
+    "1" => "573",
+    "8" => "574",
+    "14" => "575",
+    "15" => "576",
+    "19" => "577",
+    "23" => "578",
+    "24" => "622",
+    "27" => "623",
+    "2427" => "629",
+    "28" => "579",
+    "29" => "624",
+    "30" => "625",
+    "31" => "626",
+    "33" => "627",
+    "34" => "678",
+    "34E" => "679",
+    "35" => "680",
+    "36" => "681",
+    "37" => "682",
+    "38" => "683",
+    "39" => "684",
+    "40" => "685",
+    "41" => "580",
+    "42" => "581",
+    "44" => "582",
+    "45" => "583",
+    "47" => "584",
+    "50" => "686",
+    "51" => "687",
+    "66" => "585",
+    "69" => "590",
+    "71" => "591",
+    "72" => "592",
+    "73" => "594",
+    "74" => "595",
+    "75" => "596",
+    "77" => "597",
+    "77A" => "598",
+    "78" => "599",
+    "80" => "600",
+    "86" => "601",
+    "87" => "602",
+    "88" => "603",
+    "89" => "688",
+    "90" => "689",
+    "94" => "690",
+    "96" => "604",
+    "170" => "588",
+    "171" => "589",
+    "226" => "809",
+    "230" => "810",
+    "236" => "811",
+    "245" => "628"
+  }
+
+  @string_take_lookup %{
+    "the next bus to" => "543",
+    "departs" => "502",
+    "arrives" => "503",
+    "in" => "504",
+    "upcoming departures" => "548",
+    "upcoming arrivals" => "550",
+    "is now arriving" => "24055",
+    "upper level departures" => "616",
+    "lower level departures" => "617",
+    "board routes 71 and 73 on upper level" => "618",
+    "departing" => "530",
+    "arriving" => "531",
+    "," => "21012",
+    "minute" => "532",
+    "minutes" => "505"
```

Comment:
Minor: Atoms are generally preferred for static content where it makes sense, particularly as map keys. Mainly because they take up less memory and are faster for comparisons. It doesn't make as much sense for the prior two maps because one is basically all numeric and the other gets the values from the API and we don't want to convert strings to atoms especially from a dynamic source like the API. I also find that atoms are easier to read. Could we opt for atoms for this map? `,` could just be `:comma`